staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -32,6 +38,10 @@ type attributesRecord struct {
 	object      runtime.Object
 	oldObject   runtime.Object
 	userInfo    user.Info
+	annotations map[string]string
+	// other elements are always accessed in single goroutine.
+	// But ValidatingAdmissionWebhook sets annotations concurrently.
+	annotationLock sync.RWMutex
```

Comment:
nit: but on-top of annotations and add empty line above

Reply:
Done.